assets/scss/specific-brandings.scss

```diff
@@ -1,103 +1,5 @@
-/**********************
-  Uranus branding
-    Magistrates
-**********************/
-.hale-branding--uranus {
-	//header colour.
-	.govuk-header {
-		padding-bottom: 1em;
-
-		a:focus img {
-			filter: invert(1) hue-rotate(180deg);
-		}
-
-		.hale-header__logotype-text--custom {
-			font-weight: 400;
-			font-size: 22px;
-		}
-	}
-
-	.govuk-button {
-		border-radius: 4px;
-		border-width: 0;
-	}
-
-	.govuk-footer__inline-list-item {
-		display: block;
-		@include govuk-media-query($from: tablet) {
-			display: inline-block;
-		}
-	}
-}
-
-/**********************
-  Magirec new theme
-**********************/
-.hale-branding--eris .govuk-header__container {
-	border-bottom: #336C83 solid 10px;
-}
-
-.hale-branding--default,
-.hale-branding--eris {
-	.govuk-header {
-		border-bottom-width: unset;
-		.govuk-header__logo {
-			min-height: 49px;
-
-			a:focus img {
-				filter: invert(1) hue-rotate(180deg);
-			}
-		}
-	}
-	.wp-block-button__link:not(.govuk-button) {
-		@extend .govuk-button;
-	}
-	.govuk-header .hale-header__search-form button.hale-search__submit {
-		box-shadow: none;
-	}
-	.hale-search__submit:focus:not(:active):not(:hover) {
-		box-shadow: 0px 1px 0 0px govuk-colour("yellow");
-	}
-	.mojblocks-hero__image {
-		margin-top: -3px;
-	}
-}
-
-/**********************
-    Neptune branding
-          IMA
-**********************/
-.hale-branding--neptune {
-
-	.govuk-skip-link:focus {
-		outline-color: $colour_capri_teal;
-	}
-
-	.govuk-header .govuk-header__logo {
-		min-height: 49px;
-
-		a:focus img {
-			filter: invert(1) hue-rotate(180deg);
-		}
-	}
-
-	.govuk-footer__inline-list-item {
-		display: block;
-		@include govuk-media-query($from: tablet) {
-			display: inline-block;
-		}
-	}
-}
-
-/**********************
-    Venus branding
-        VC
-**********************/
-.hale-branding--venus {
-	.govuk-header .govuk-header__logo {
-
-		a:focus img {
-			filter: invert(1) hue-rotate(180deg);
-		}
+.mojblocks-hero {
```

Comment:
Can we move this so we can remove this sass file

Reply:
Done - moved to moj-blocks-general.scss